apps/calendar/js/views/month_day.js

```diff
@@ -45,6 +45,7 @@ MonthDay.prototype = {
     );
     el.dataset.date = dayId;
     el.className = state;
+    el.classList.add('month-day');
```

Comment:
`state` can be like `"past other-month"` or `"present"` or `"future"` or `"future other-month"` so `el.classList.add(state)` would fail (space is not a valid char..)